corehq/ex-submodules/pillowtop/migrations/0007_update_user_mapping_with_assigned_location_ids.py

```diff
@@ -0,0 +1,28 @@
+
+from django.db import migrations
+
+from corehq.pillows.mappings.user_mapping import USER_INDEX_INFO
+from corehq.util.django_migrations import update_es_mapping
```

Comment:
Decorator needed for suggestion below.
```suggestion
from corehq.util.django_migrations import skip_on_fresh_install, update_es_mapping
```